devtools/x/src/nextest.rs

```diff
@@ -24,6 +26,13 @@ use structopt::StructOpt;
 
 #[derive(Debug, StructOpt)]
 pub struct Args {
+    /// Nextest config options
+    #[structopt(flatten)]
+    config_opts: ConfigOpts,
+    /// Nextest profile to use
+    #[structopt(long, short = "P")]
```

Comment:
I think for testing tools sane defaults should need to be only very rarely overridden.   I'd like us to not need a profile.   Can you describe it's use cases?

Reply:
The specific use case I had in mind was having different settings for local and CI runs (local runs don't do retries, CI runs do do them, for example), but also make it possible for local developers to emulate CI easily.